app/views.py

```diff
@@ -101,11 +102,12 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ContractsTableView(LoginRequiredMixin, SingleTableMixin, FilterView):
+class ContractsTableView(LoginRequiredMixin, SingleTableMixin, PaginationMixin, FilterView):
     queryset = Contract.objects.all()
     table_class = ContractsTable
     template_name = "app/contracts_table.html"
     filterset_class = ContractsFilter
+    paginate_by = 15
```

Comment:
Maybe we could use a constant here, so we reuse it in our tests and make it consistent across all views.